saltproc/saltproc.py

```diff
@@ -374,6 +383,7 @@ def reopen_db(self, restart):
             # set past time
             # !! this time thing should be made certain
             self.current_step = np.amax(np.nonzero(self.keff)) + 1
+            self.current_step = np.amax(np.nonzero(sum(self.driver_before_db))) + 1
```

Comment:
Here you're setting a variable equal to something and then set it equal to something else in the next line.
Delete one of the two. Ideally, simplifying the syntax.